ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,209 @@
+2. Shows the skull-stripped brain (green) overlaid on the original t1w (magenta)
+3. save the image into output directory
+"""
+
+import warnings
+
+warnings.simplefilter("ignore")
+import os
+import re
+import sys
+import numpy as np
+import nibabel as nb
+from argparse import ArgumentParser
+from scipy import ndimage
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib as mpl
+
+mpl.use("Agg")  # very important above pyplot import
+from nilearn.plotting.edge_detect import _edge_map as edge_map
+import matplotlib.pyplot as plt
+from ndmg.utils.qa_utils import get_min_max, opaque_colorscale, pad_im
+from ndmg.utils.gen_utils import get_filename, get_braindata
+
+
+
+
+
+def gen_overlay_pngs(
+    brain, original, outdir, loc=0, mean=False, minthr=2, maxthr=95, edge=False):
+    """Generate a QA image for skullstrip.
+    will call the function plot_overlays_skullstrip
+
+    Parameters
+    ----------
+    brain: nifti file
+        Path to the skull-stripped nifti brain
+    original: nifti file
+        Path to the original t1w brain, with the skull included
+    outdir: str
+        Path to the directory where QA will be saved
+    loc: int
+    mean: bool
+    minthr: int
+    maxthr: int
+    edge: bool
+    """
+    original_name = get_filename(original)
+    brain_data = nb.load(brain).get_data()
+    if brain_data.ndim == 4:  # 4d data, so we need to reduce a dimension
+        if mean:
+            brain_data = brain_data.mean(axis=3)
+        else:
+            brain_data = brain_data[:, :, :, loc]
+
+
+    cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["white", "magenta"])
+    cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["white", "green"])
+
+
+    fig = plot_overlays_skullstrip(brain_data, original)
+
+    # name and save the file
+    fig.savefig(f"{outdir}/qa_skullstrip__{original_name}.png", format="png")
+
+
+
+
+
+def plot_overlays_skullstrip(brain, original, cmaps=None, minthr=2, maxthr=95, edge=False):
+    """Shows the skull-stripped brain (green) overlaid on the original t1w (magenta)
+
+    Parameter
+    ---------
+    brain: str, nifti image, numpy.ndarray
+        an object to open the data for a skull-striped brain. Can be a string (path to a brain file),
+        nibabel.nifti1.nifti1image, or a numpy.ndarray.
+​
+    original: str, nifti image, numpy.ndarray
+        an object to open the data for t1w brain, with the skull included. Can be a string (path to a brain file),
+        nibabel.nifti1.nifti1image, or a numpy.ndarray.
+​
+    cmap: Colormap objects based on lookup tables using linear segments.
+    minthr: int
+    maxthr: int
+    edge: bool
+​
+    Returns
+    ---------
+    foverlay: matplotlib.figure.Figure
+
+    """
+
+    plt.rcParams.update({"axes.labelsize": "x-large", "axes.titlesize": "x-large"})
+    foverlay = plt.figure()
+
+    original = get_braindata(original)
+    ori_shape = get_braindata(brain).shape
```

Comment:
Change `ori_shape` to `brain_shape`, it's is confusing to have ori_shape not belong to `original`

Reply:
https://github.com/neurodata/ndmg/pull/366/commits/65d670a04f209da4901ad6c0799cce039c700dcb